server/publicapi/packages/service.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+import logging
+from superdesk.services import BaseService
+
+
+logger = logging.getLogger(__name__)
+
+
+class PackagesService(BaseService):
+    """
+    Data layer providing the CRUD functionality for the `package`
```

Comment:
Yeah, I was contemplating what term to use here. Suggestions welcome, because a misleading comment is even worse than no comment at all.